teacher.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'person'
+
+class Teacher < Person
+  def initialize(age, specialization, name: 'Unknown', parent_permission: true)
+    super(age, parent_permission, name)
```

Comment:
- [ ] Kindly use the colon notation to assign the `name` and `parent_permission` parameters to avoid ArgumentError as you already use the colon notation in the parent class(Person), also stick with the order of parameters in the parent class as the `name` parameter comes before `parent_permission`, so it will be like:
```ruby
super(age, name: name, parent_permission: parent_permission)
```

Reply:
@gemmen29 fixed